run

```diff
@@ -141,20 +168,33 @@ docker_django () {
 docker_yarn () {
   # Run "yarn" from the "node" image
   docker_run  \
+    ${yarn_cache_arg} \
     ${module_volumes[@]+"${module_volumes[@]}"}  `# Add any override modules as volumes`  \
-    $yarn_image $@               `# Use the image for node version 7`
+    $yarn_image $@              `# Use the image for node version 7`
 }
 
 yarn_install () {
   # Install yarn dependencies, without module overrides
   docker_run $yarn_image install
 }
 
+clean_cache () {
+  # Clean yarn cache volume
+  echo "Deleting yarn_cache Docker volume..."
+  docker volume rm yarn_cache 2>/dev/null || echo "No volume found"
+  if [ $LOCAL_YARN_INSTALLED = true ]; then
+    if [ $OPT_CACHE_CLEAN_FORCE = true ]; then
+      yarn cache clean
+    else
+      echo "#"
+      echo "# Yarn is locally installed, use --force to clean your local cache"
```

Comment:
@WillMoggridge if you're getting sick of the amount of discussion going on here, I'm also happy for you to merge this and then I can propose my ideas as a future PR.